manifests/upstream/template-operator/base/kustomization.yaml

```diff
@@ -24,7 +24,7 @@ patchesStrategicMerge:
               resources:
                 limits:
                   cpu: 150m
-                  memory: 200Mi
+                  memory: 2Gi
                 requests:
                   cpu: 100m
-                  memory: 150Mi
+                  memory: 1Gi
```

Comment:
I don't think we can do this by default - a 200Mi request and 1Gi limit should be sufficient, cpu limit can also be bumped to 500m

Reply:
ok.  I thought these were the numbers you mentioned in the meeting, but I can adjust them